src/plugins/extras/README.md

```diff
@@ -0,0 +1,26 @@
+Extras Plugin
```

Comment:
Oh right: The extra addons themselves aren't yet in this branch. If you want to see some of them in action, see http://henrikingo.github.io/impress.js/examples/classic-slides/#/step-1

The point of this plugin is to simplify and standardize their use, so that if, for example, markdown.js was added with a script tag, then this plugin detects it and does whatever is necessary to convert markdown to html. The uses doesn't need to add additional `init()` or other lines of code to make it happen. In this sense, even if extra addons aren't part of impress.js, this still defines a small set of extra libraries who's use is supported and standardized.